src/java.base/share/native/libzip/zlib/README

```diff
@@ -1,9 +1,9 @@
 ZLIB DATA COMPRESSION LIBRARY
 
-zlib 1.2.11 is a general purpose data compression library.  All the code is
+zlib 1.2.13 is a general purpose data compression library.  All the code is
 thread safe.  The data format used by the zlib library is described by RFCs
 (Request for Comments) 1950 to 1952 in the files
-https://tools.ietf.org/html/rfc1950 (zlib format), rfc1951 (deflate format) and
+http://tools.ietf.org/html/rfc1950 (zlib format), rfc1951 (deflate format) and
```

Comment:
Hello Lance, it appears that we had done this change to this file in the JDK as part of https://bugs.openjdk.org/browse/JDK-8283525. As Sean notes in that issue, the `http` link is back to working but `https` was still prefered during that change. Should we stick with `https`?

Reply:
Hi Jai,

Given this is how it is in the README in the upstream repository and it is accessible, I would prefer not to change it and when/if we do, we should have the change made upstream

Reply:
Thank you Lance, that sounds fine to me.